src/QMCWaveFunctions/SpinorSet.h

```diff
@@ -14,6 +14,7 @@
 #define QMCPLUSPLUS_SPINORSET_H
 
 #include "QMCWaveFunctions/SPOSet.h"
+#include "VirtualParticleSet.h"
```

Comment:
Please move this include to the cpp file. Also the test file will need this include.

Reply:
actually, this is already included in SPOSet, which is included. So I think I can just remove this altogether. For some reason, my editor thought it was needed but it doesn't